cmd/createcluster_internal_test.go

```diff
@@ -145,3 +147,12 @@ func TestValidNetwork(t *testing.T) {
 	err = validateClusterConfig(conf)
 	require.NoError(t, err)
 }
+
+func TestInvalidThreshold(t *testing.T) {
+	conf := clusterConfig{
+		NumNodes:  5,
+		Threshold: 3,
+	}
+	err := validateClusterConfig(conf)
+	require.EqualError(t, err, "threshold less than minimum")
```

Comment:
also test if we get `z.Int("minimum", 4)`

Reply:
done